includes/class-fontawesome.php

```diff
@@ -1363,8 +1370,25 @@ function( $html, $handle ) use ( $resource_collection ) {
 				);
 
 				if ( $load_spec['v4shim'] ) {
-					// phpcs:ignore WordPress.WP.EnqueuedResourceParameters
-					wp_enqueue_style( self::RESOURCE_HANDLE_V4SHIM, $resource_collection[1]->source(), null, null );
+					add_action(
+						'wp_enqueue_scripts',
+						function () use ( $resource_collection, $options, $license_subdomain ) {
+						// phpcs:ignore WordPress.WP.EnqueuedResourceParameters
+							wp_enqueue_style( self::RESOURCE_HANDLE_V4SHIM, $resource_collection[1]->source(), null, null );
+
+							wp_add_inline_style(
+								self::RESOURCE_HANDLE_V4SHIM,
+								"@font-face {\n" .
+								"    font-family: \"FontAwesome\";\n" .
+								"    font-style: normal;\n" .
+								"    font-weight: 900;\n" .
+								"    src: url(\"https://${license_subdomain}.fontawesome.com/releases/v" . $options['version'] . "/webfonts/fa-solid-900.eot\");\n" .
```

Comment:
@robmadole This copies pretty closely what we put into `all.css`.
In customer scenarios, having only the `src: url` for the `.eot` file did not work. It worked after adding the `.woff2`.
Is there a more minimal set of these that should be included here?